canonical-web-and-design/maas-ui
# Serve React and AngularJs clients together

Done:
- Add an app to serve the React and AngularJs with a proxy. Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/279.
- Remove redundant files from the UI project (env settings etc.)

QA:
- Remove your ./ui/.env.development.local.
- copy ./legacy/.env to ./legacy/.env.local and set the host to your MAAS.
- copy ./proxy/.env to ./proxy/.env.local and set the url to your MAAS.
- Run `./run`.
- Visit <your-multipass>:8400/MAAS/r/ and login (you may need to clear your sessionid and csrftoken cookies).
- You should see the React app.
- Using the header click on an item that isn't in the React app e.g. Controllers
- You should switch to the Angular app.
